jshell.support/src/org/netbeans/modules/jshell/tool/JShellLauncher.java

```diff
@@ -249,7 +249,7 @@ protected JShell createJShellInstance() {
             Thread.currentThread().setContextClassLoader(Lookup.getDefault().lookup(ClassLoader.class));
             JShell.Builder b = makeBuilder();
             if (execGen != null) {
-                    b.executionEngine(new CaptureExecControl(execGen), Collections.emptyMap());
+                    b.executionEngine(new CaptureExecControl(execGen), null);
```

Comment:
There's also another overload that does not take the second param that (I guess) has the same meaning. But this is not a big deal.